### DISCUSSION TITLE
How to specify Anki tags

### DISCUSSION THREAD

**Discussion by paulcarey** [debanjandhar12/logseq-anki-sync#123] [General]
Hi, is it possible to specify tags within Logseq so they're populated in the Anki tags field? I've tried a bunch of different syntaxes but none worked, and didn't see this described in the docs either.

**Comment by debanjandhar12** [comment]
Tags are now supported in V4. See #110

**Comment by thiswillbeyourgithub** [comment]
Update: since V4.6.1 there are new ways to specify tags with properties. See issue https://github.com/debanjandhar12/logseq-anki-sync/issues/147#issuecomment-1518644452

Thanks again @debanjandhar12  !
